src/Apps/Artwork/Components/ImageCarousel.tsx

```diff
@@ -36,8 +37,9 @@ const SmallImageArea = styled(BaseImageArea)`
 `
 
 const LargeImageArea = styled(BaseImageArea)`
-  min-height: 450px;
-  height: calc(100vh - 160px);
+  /* FIXME: Is this still needed? */
```

Comment:
This would lead to overflows, and when there are multiple images the arrows would shift. Not sure exactly why, but wanted to leave this in while things settle a bit.